lesson2/recur2.py

```diff
@@ -0,0 +1,12 @@
+def st2(x):
```

Comment:
If function returns bool then is should start with 'is'
```suggestion
def is_st2(x):
```